js/app.js

```diff
@@ -0,0 +1,83 @@
+
+//question 1
+
+let obj ={
+	insideObj: {
+		key1: 'yo';
+		key2: 12
+	}
+
+}
+
+console.log(insideObj.key1())
+
+// question 2
+
+let obj = {
+	 {
+		someObject: {
+			someProperty: 'hello', 'hi', 'how are you'
+ 		}
+	}
+ }
+
+ // question 3
+
+ let obj = {
+ 	property () => { ['dare', 'james']
+ 	}
+ }
+
+ // question 4
+
+ let fruits = ['james', 'milton', 'marcus':{height: 6, age: 23, }, 'david']
+
+ console.log(fruits[2].())
```

Comment:
getting there... on line 33 either get rid of marcus and the comma after the age property, or enclose the entire thing, including marcus in another set of {} like this:
`let fruits = ['james', 'milton', {height: 6, age: 23}, 'david']`
or this
`let fruits = ['james', 'milton', {'marcus':{height: 6, age: 23 } }, 'david'] `
and then on line 35 your syntax is not right.  you can't do .() there (or at all)